content/gherkin.md

```diff
@@ -226,6 +229,8 @@ more `Given` steps.
 
 A `Background` is run before *each* scenario, but after any [Before hooks](/cucumber/#hooks). In your feature file, put the `Background` before the first `Scenario`.
 
+You can only have one set of `Background` steps per feature. If you need different `Background` steps for different features, then it's indicative of you needing to isolate your test cases into 2 or more features.
```

Comment:
"for different features, then it's indicative of you needing to isolate your test cases into 2 or more features." ->
"for different scenarios, you'll need to split them into different feature files." (shorter/clearer)